app/src/main/java/org/fossasia/openevent/events/NoInternetEvent.java

```diff
@@ -0,0 +1,7 @@
+package org.fossasia.openevent.events;
+
+/**
+ * Created by MananWason on 8/4/2015.
+ */
+public class NoInternetEvent {
```

Comment:
Is upto you, you can pass the entire retrofit error as a parameter to an
event and let the UI decide what to display or you can fire events by
yourself, your choice but I don't want to see failure message about
"internet connection" when my internet connection is good but server is
down.

On Mon, Aug 10, 2015 at 3:06 AM, Manan Wason <notifications@github.com>
wrote:

> In app/src/main/java/org/fossasia/openevent/events/NoInternetEvent.java
> <https://github.com/fossasia/open-event-android/pull/69#discussion_r36592389>
> :
>
> > @@ -0,0 +1,7 @@
> > +package org.fossasia.openevent.events;
> > +
> > +/**
> > + * Created by MananWason on 8/4/2015.
> > + */
> > +public class NoInternetEvent {
>
> So what do you suggest? We should handle all these error messages or just
> give a common one for errors and another one for internet not connected?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/fossasia/open-event-android/pull/69/files#r36592389>.
>



-- 
Mohit Singh KANWAL (Mr)

[image: Facebook] <http://facebook.com/mohitkanwal>  [image: Twitter]
<http://twitter.com/mohitkanwal>